src/main/resources/application.yaml

```diff
@@ -102,12 +102,11 @@ spring:
               create-source: metadata
 
   # for microservices with elasticsearch
-  data:
-    elasticsearch:
-      host: elasticsearch
-      port: 9200
-      client:
-        timeout: 60  # Seconds
+  elasticsearch:
+    uris:
+      - http://elasticsearch:9200
+    socket-timeout: 60s
+    connection-timeout: 60s
```

Comment:
```suggestion

```
We should remove the connection timeout even if it was present in the previous configuration. We saw with @SlimaneAmar that it's not expected. The socket timeout is here to be able to reindex studies completely (see PR that Kevin found) and should be kept.